### DISCUSSION TITLE
Normal component E-Fields

### DISCUSSION THREAD

**Discussion by robertorgarcia** [simnibs/simnibs#386] [9. Results and analysis]
Hi, 

I've been working on tDCS simulations using SimNIBS and am currently analyzing the results. To assess the E-field behavior on the cortical surface of the brain, I utilized your code for mesh interpolation (specifically, trans.middle_gm_interpolation). According to the principles from SimNIBS simulations and assuming our system is closed, the sum of all normal components of the E-field across the entire cortical surface should ideally equal zero.

However, upon reviewing my simulations, I consistently find that the total normal component does not equate to zero or near to zero. This discrepancy prompts me to seek insights into why this might be occurring.

I appreciate any suggestions or thoughts on how to interpret this finding accurately.

Thanks,

**Comment by Kstoffer** [comment]
The simulations will calculate the potential at the tetrahedra, and converting this into the electric field at nodes will require evaluation of both the gradient and an interpolation. Therefore, there is no guarantee that the normal component will sum to zero at the surfaces as errors might accumulate. If you increase the density of the mesh the errors are expected to decrease.
Best,
Kristoffer
